superset/assets/src/explore/components/controls/DateFilterControl.jsx

```diff
@@ -346,13 +348,14 @@ export default class DateFilterControl extends React.Component {
       ));
     const timeFrames = COMMON_TIME_FRAMES.map((timeFrame) => {
       const nextState = getStateFromCommonTimeFrame(timeFrame);
+      const endpoints = this.props.endpoints;
       return (
         <OverlayTrigger
           key={timeFrame}
           placement="left"
           overlay={
             <Tooltip id={`tooltip-${timeFrame}`}>
-              {nextState.since}<br />{nextState.until}
+              {nextState.since} {endpoints ? `(${endpoints[0]})` : ''}<br />{nextState.until} {endpoints ? `(${endpoints[1]})` : ''}
```

Comment:
these can be `endpoints && `(${endpoints[0]})` instead of the ternary with the empty string